doc/release/1.11.0-notes.rst

```diff
@@ -25,9 +32,19 @@ Build System Changes
 Future Changes
 ==============
 
-* Relaxed stride checking will become the default in 1.12.0.
-* Support for Python 2.6, 3.2, and 3.3 will be dropped in 1.12.0.
-* ``MaskedArray`` takes view of data **and** mask when slicing in 1.12.0.
+The following changes are scheduled for Numpy 1.12.0.
+
+* Relaxed stride checking will become the default.
+* Support for Python 2.6, 3.2, and 3.3 will be dropped.
+* Slicing a ``MaskedArray`` will return views of both data **and** mask.
+* NaT will always compare False except for NaT != NaT, which will be True.
+
+In a future release the following changes will be made.
+
+* The ``rand`` function exposed in ``numpy.testing`` will be removed. Use
+  the random number generators from ``numpy.random`` instead.
+* Views that change the dimensions of an array will only work for
+  C_CONTIGUOUS arrays and will only affect the last dimension.
```

Comment:
Instead of "views" this should say "calls to `ndarray.view(dtype=new_dtype)`" or something -- we certainly will continue to allow for views that change the shape of an array (e.g. via `reshape`) :-)